components/aiprompt/templates.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Templates
+page_title: AIPrompt Templates
+description: Customize the appearance of the AIPrompt through its templates.
+slug: aiprompt-templates
+tags: telerik,blazor,aiprompt,ai,prompt,templates
+published: True
+position: 30
+---
+
+# AIPrompt Views Templates
+
+This article explains the available templates of the AIPrompt component. For templating an entire view, refer to the [AIPrompt Views]({%slug aiprompt-templates%}) article.
+
+- [AIPrompt Views Templates](#aiprompt-views-templates)
+  - [PromptSuggestionItemTemplate](#promptsugestionitemtemplate)
+  - [CommandItemTemplate](#commanditemtemplate)
+
+
+## PromptSuggestionItemTemplate
+
+The Prompt View of the AIPrompt renders any suggestions passed to the `PromptSuggestions` parameter in the form of elevated bubbles within a collapsible section. The `PromptSuggestionItemTemplate` allows you to control the rendering of individual suggestions.
+
+>note By default, user click on a suggestion will result in the prompt's input being populated with the suggestion's value. Such an interaction will also trigger a `PromptTextChanged` event. If you utilize the `PromptSuggestionItemTemplate`, you should also handle any event you deem necessary(such as `onclick`).
+
+>caption Use the `PromptSuggestionItemTemplate` to alter the appearance of the suggestions
+
+````CSHTML
+<TelerikAIPrompt @bind-Prompt="@Prompt" PromptSuggestions="@Suggestions">
+    <PromptSuggestionItemTemplate>
+        <div @onclick="@OnSuggestionClick" class="my-custom-suggestion-item">
+            <TelerikSvgIcon Icon="@SvgIcon.Clipboard" />
+            <span class="my-custom-suggestion-item-text">@context</span>
+        </div>
+    </PromptSuggestionItemTemplate>
+</TelerikAIPrompt>
+
+<style>
+    .my-custom-suggestion-item {
+        display: flex;
+        align-items: center;
+        width: fit-content;
+        padding: 0.5em 1.5em;
+        border-radius: 20px;
+        box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+        cursor: pointer;
+        transition: transform 0.2s ease-in-out;
+    }
+
+    .my-custom-suggestion-item:hover {
+        transform: scale(1.05);
+    }
+
+    .my-custom-suggestion-item-text {
+        padding-inline-start: 0.5em;
+    }
+    
+</style>
+
+@code {
+    public string Prompt { get; set; }
+    public TelerikAIPrompt AIPromptRef { get; set; }
+
+    public List<string> Suggestions { get; set; } = new List<string>()
+    {
+        "Explain quantum physics in simple terms.",
+        "What are the three laws of thermodynamics?"
+    };
+
+    private void OnSuggestionClick()
+    {
+        // handle suggestion click here.
+    }
+}
+````
+
+## CommandItemTemplate
+
+The `CommandItemTemplate` allows you to override the rendering of individual command items rendered within the Commands View.
```

Comment:
```suggestion
The `CommandItemTemplate` allows you to override the rendering of individual command items in the Commands View.
```

Reply:
This template will not be included in v1, thus this entire section will be omitted.